prow/plugins/override/override_test.go

```diff
@@ -361,6 +426,7 @@ func TestHandle(t *testing.T) {
 					State:       github.StatusSuccess,
 				},
 			},
+			usesAppsAuth: true,
```

Comment:
Intended?

Reply:
yes, want to test if the new code modifies existing logic